.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "\U0001F4DC Documentation"
+description: Suggest an idea for this project
+title: 'DOCS: '
+labels: [documentation]
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what documentation is missing or incorrect.
+
+**Where to find**
+Where the error is seen or where documentation shoud be:
+(website/github/docstrings/etc.)
+- Make a list
```

Comment:
"Location" instead of "Where to find"? Also typo "should". Just generally a bit unclear.